windows/Dockerfile

```diff
@@ -0,0 +1,87 @@
+ARG UBUNTU_VERSION=20.04
+
+FROM ubuntu:${UBUNTU_VERSION} AS full
+
+RUN echo 'APT::Install-Recommends 0;' >> /etc/apt/apt.conf.d/01norecommends \
+  && echo 'APT::Install-Suggests 0;' >> /etc/apt/apt.conf.d/01norecommends \
+  && apt-get update \
+  && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    ca-certificates apt-transport-https curl \
+    xorriso mtools dosfstools qemu-utils qemu-kvm \
+  && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /app
+
+# IDA Freeware: only permissible for free and open deployments of the dojo!
+ARG INSTALL_IDA=False
```

Comment:
No IDA

Reply:
As in, remove IDA since the linux one is sufficient?

Reply:
Oh right, I forgot you install IDA on windows as well. Disregard!